WARNING 2019-07-10 03:03:42,301 log --> Not Found: /api/v2/login/
WARNING 2019-07-10 03:03:42,301 log --> Not Found: /api/v2/login/
WARNING 2019-07-10 03:03:42,302 basehttp --> "GET /api/v2/login/ HTTP/1.1" 404 2493
INFO 2019-07-10 03:03:53,079 basehttp --> "GET /api/v1/balancesheet/ HTTP/1.1" 200 11063
INFO 2019-07-10 03:03:53,176 basehttp --> "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2019-07-10 03:03:53,177 basehttp --> "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2019-07-10 03:03:53,221 basehttp --> "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2019-07-10 03:03:53,221 basehttp --> "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2019-07-10 03:03:53,232 basehttp --> "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2019-07-10 03:03:53,244 basehttp --> "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2019-07-10 03:03:53,245 basehttp --> "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2019-07-10 03:03:53,247 basehttp --> "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2019-07-10 03:03:53,251 basehttp --> "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2019-07-10 03:03:53,275 basehttp --> "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2019-07-10 03:03:53,316 basehttp --> "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
WARNING 2019-07-10 03:03:53,466 log --> Not Found: /favicon.ico
WARNING 2019-07-10 03:03:53,466 log --> Not Found: /favicon.ico
WARNING 2019-07-10 03:03:53,466 basehttp --> "GET /favicon.ico HTTP/1.1" 404 2487
INFO 2019-07-10 03:06:48,818 basehttp --> "GET /api/v1/balancesheet/1 HTTP/1.1" 200 9884
INFO 2019-07-10 03:06:54,406 basehttp --> "GET /api/v1/balancesheet/2 HTTP/1.1" 200 9884
INFO 2019-07-10 03:06:57,602 basehttp --> "GET /api/v1/balancesheet/3 HTTP/1.1" 200 9881
INFO 2019-07-10 03:11:40,063 basehttp --> "GET /api/v1/balancesheet/3 HTTP/1.1" 200 9881
INFO 2019-07-10 03:11:43,162 basehttp --> "GET /api/v1/balancesheet/4 HTTP/1.1" 200 9881
WARNING 2019-07-10 03:34:40,399 log --> Not Found: /api/v1/balancesheet/7
WARNING 2019-07-10 03:34:40,399 log --> Not Found: /api/v1/balancesheet/7
WARNING 2019-07-10 03:34:40,400 basehttp --> "GET /api/v1/balancesheet/7 HTTP/1.1" 404 9665
WARNING 2019-07-10 07:13:18,870 log --> Not Found: /api/v2/login/
WARNING 2019-07-10 07:13:18,870 log --> Not Found: /api/v2/login/
WARNING 2019-07-10 07:13:18,870 basehttp --> "GET /api/v2/login/ HTTP/1.1" 404 2493
WARNING 2019-07-10 10:52:44,338 log --> Not Found: /api/v2/login/
WARNING 2019-07-10 10:52:44,338 log --> Not Found: /api/v2/login/
WARNING 2019-07-10 10:52:44,356 basehttp --> "GET /api/v2/login/ HTTP/1.1" 404 2493
WARNING 2019-07-11 17:01:30,835 log --> Not Found: /api/v2/login/
WARNING 2019-07-11 17:01:30,835 log --> Not Found: /api/v2/login/
WARNING 2019-07-11 17:01:30,836 basehttp --> "GET /api/v2/login/ HTTP/1.1" 404 2493
WARNING 2019-07-11 17:01:35,902 log --> Not Found: /
WARNING 2019-07-11 17:01:35,902 log --> Not Found: /
WARNING 2019-07-11 17:01:35,903 basehttp --> "GET / HTTP/1.1" 404 2436
WARNING 2019-07-11 17:01:35,974 log --> Not Found: /favicon.ico
WARNING 2019-07-11 17:01:35,974 log --> Not Found: /favicon.ico
WARNING 2019-07-11 17:01:35,975 basehttp --> "GET /favicon.ico HTTP/1.1" 404 2487
WARNING 2019-07-11 17:01:50,950 log --> Not Found: /api/v1/balancesheet/
WARNING 2019-07-11 17:01:50,950 log --> Not Found: /api/v1/balancesheet/
WARNING 2019-07-11 17:01:50,951 basehttp --> "GET /api/v1/balancesheet/ HTTP/1.1" 404 2614
WARNING 2019-07-11 17:01:55,432 log --> Not Found: /api/v1/balancesheet/1
WARNING 2019-07-11 17:01:55,432 log --> Not Found: /api/v1/balancesheet/1
WARNING 2019-07-11 17:01:55,433 basehttp --> "GET /api/v1/balancesheet/1 HTTP/1.1" 404 9665
INFO 2019-07-11 17:01:55,487 basehttp --> "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2019-07-11 17:01:55,495 basehttp --> "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2019-07-11 17:01:55,496 basehttp --> "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2019-07-11 17:01:55,499 basehttp --> "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2019-07-11 17:01:55,549 basehttp --> "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2019-07-11 17:01:55,561 basehttp --> "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2019-07-11 17:01:55,566 basehttp --> "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2019-07-11 17:01:55,567 basehttp --> "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2019-07-11 17:01:55,569 basehttp --> "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2019-07-11 17:01:55,571 basehttp --> "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2019-07-11 17:01:55,663 basehttp --> "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
INFO 2019-07-11 17:02:30,618 basehttp --> "GET /api/v1/balancesheet/109 HTTP/1.1" 200 9898
INFO 2019-07-11 17:11:17,336 basehttp --> "GET /api/v1/balancesheet/109 HTTP/1.1" 200 9898
WARNING 2019-07-12 11:48:00,253 log --> Not Found: /api/v2/login/
WARNING 2019-07-12 11:48:00,253 log --> Not Found: /api/v2/login/
WARNING 2019-07-12 11:48:00,254 basehttp --> "GET /api/v2/login/ HTTP/1.1" 404 2493
WARNING 2019-07-12 11:48:39,928 log --> Not Found: /api/v1/short-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:48:39,928 log --> Not Found: /api/v1/short-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:48:39,929 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 404 2884
WARNING 2019-07-12 11:48:39,978 log --> Not Found: /favicon.ico
WARNING 2019-07-12 11:48:39,978 log --> Not Found: /favicon.ico
WARNING 2019-07-12 11:48:39,979 basehttp --> "GET /favicon.ico HTTP/1.1" 404 2487
WARNING 2019-07-12 11:48:43,100 log --> Not Found: /api/v1/short-balance-sheet/hdfc_bank/
WARNING 2019-07-12 11:48:43,100 log --> Not Found: /api/v1/short-balance-sheet/hdfc_bank/
WARNING 2019-07-12 11:48:43,100 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank/ HTTP/1.1" 404 2887
WARNING 2019-07-12 11:49:01,796 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank/
WARNING 2019-07-12 11:49:01,796 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank/
WARNING 2019-07-12 11:49:01,797 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank/ HTTP/1.1" 404 2890
WARNING 2019-07-12 11:49:17,344 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:17,344 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:17,345 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
WARNING 2019-07-12 11:49:27,061 log --> Not Found: /api/v1/sshort-balance-sheet/'hdfc_bank'
WARNING 2019-07-12 11:49:27,061 log --> Not Found: /api/v1/sshort-balance-sheet/'hdfc_bank'
WARNING 2019-07-12 11:49:27,061 basehttp --> "GET /api/v1/sshort-balance-sheet/'hdfc_bank' HTTP/1.1" 404 2917
WARNING 2019-07-12 11:49:33,287 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:33,287 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:33,288 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
WARNING 2019-07-12 11:49:49,612 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:49,612 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:49,612 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
WARNING 2019-07-12 11:49:51,227 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:51,227 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:49:51,227 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
WARNING 2019-07-12 11:50:03,314 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:03,314 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:03,315 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
WARNING 2019-07-12 11:50:05,503 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:05,503 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:05,503 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
WARNING 2019-07-12 11:50:07,934 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:07,934 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:07,935 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
WARNING 2019-07-12 11:50:10,796 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank/
WARNING 2019-07-12 11:50:10,796 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank/
WARNING 2019-07-12 11:50:10,796 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank/ HTTP/1.1" 404 2890
WARNING 2019-07-12 11:50:13,953 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:13,953 log --> Not Found: /api/v1/sshort-balance-sheet/hdfc_bank
WARNING 2019-07-12 11:50:13,954 basehttp --> "GET /api/v1/sshort-balance-sheet/hdfc_bank HTTP/1.1" 404 2887
ERROR 2019-07-12 11:50:24,143 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:50:24,143 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:50:24,145 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125812
ERROR 2019-07-12 11:52:18,307 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:18,307 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:18,308 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:52:20,122 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:20,122 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:20,122 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:52:21,736 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:21,736 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:21,737 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:52:23,405 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:23,405 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:23,405 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:52:43,972 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:43,972 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:43,975 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:52:45,801 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:45,801 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:45,802 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:52:47,517 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:47,517 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:52:47,517 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:54:25,076 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:25,076 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:25,077 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:54:26,475 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:26,475 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:26,476 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:54:31,389 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:31,389 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:31,389 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:54:32,793 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:32,793 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:32,793 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:54:44,169 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:44,169 log --> Internal Server Error: /api/v1/short-balance-sheet/hdfc_bank
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/db/models/query.py", line 403, in get
    (self.model._meta.object_name, num)
stocks_detail_api.models.ShortBalanceSheetModel.MultipleObjectsReturned: get() returned more than one ShortBalanceSheetModel -- it returned 5!
ERROR 2019-07-12 11:54:44,170 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 500 125947
ERROR 2019-07-12 11:56:14,730 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 94, in get_object
    (self.__class__.__name__, lookup_url_kwarg)
AssertionError: Expected view ShortBalanceSheetApi to be called with a URL keyword argument named "pk". Fix your URL conf, or set the `.lookup_field` attribute on the view correctly.
ERROR 2019-07-12 11:56:14,730 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 94, in get_object
    (self.__class__.__name__, lookup_url_kwarg)
AssertionError: Expected view ShortBalanceSheetApi to be called with a URL keyword argument named "pk". Fix your URL conf, or set the `.lookup_field` attribute on the view correctly.
ERROR 2019-07-12 11:56:14,731 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 500 106941
ERROR 2019-07-12 11:57:05,187 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 241, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 47, in list
    serializer = self.get_serializer(queryset, many=True)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 112, in get_serializer
    return serializer_class(*args, **kwargs)
TypeError: 'ListSerializer' object is not callable
ERROR 2019-07-12 11:57:05,187 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 241, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 47, in list
    serializer = self.get_serializer(queryset, many=True)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 112, in get_serializer
    return serializer_class(*args, **kwargs)
TypeError: 'ListSerializer' object is not callable
ERROR 2019-07-12 11:57:05,188 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 500 109217
ERROR 2019-07-12 11:57:06,680 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 241, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 47, in list
    serializer = self.get_serializer(queryset, many=True)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 112, in get_serializer
    return serializer_class(*args, **kwargs)
TypeError: 'ListSerializer' object is not callable
ERROR 2019-07-12 11:57:06,680 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 241, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 47, in list
    serializer = self.get_serializer(queryset, many=True)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 112, in get_serializer
    return serializer_class(*args, **kwargs)
TypeError: 'ListSerializer' object is not callable
ERROR 2019-07-12 11:57:06,681 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 500 109217
INFO 2019-07-12 11:57:18,939 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 28172
INFO 2019-07-12 11:57:19,050 basehttp --> "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,052 basehttp --> "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,072 basehttp --> "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,072 basehttp --> "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,073 basehttp --> "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,129 basehttp --> "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,151 basehttp --> "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,248 basehttp --> "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,252 basehttp --> "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,252 basehttp --> "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
INFO 2019-07-12 11:57:19,272 basehttp --> "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
INFO 2019-07-12 11:57:28,321 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 28172
INFO 2019-07-12 11:57:29,609 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 28172
ERROR 2019-07-12 11:57:37,727 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 94, in get_object
    (self.__class__.__name__, lookup_url_kwarg)
AssertionError: Expected view ShortBalanceSheetApi to be called with a URL keyword argument named "pk". Fix your URL conf, or set the `.lookup_field` attribute on the view correctly.
ERROR 2019-07-12 11:57:37,727 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 210, in get
    return self.retrieve(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 94, in get_object
    (self.__class__.__name__, lookup_url_kwarg)
AssertionError: Expected view ShortBalanceSheetApi to be called with a URL keyword argument named "pk". Fix your URL conf, or set the `.lookup_field` attribute on the view correctly.
ERROR 2019-07-12 11:57:37,728 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 500 107076
INFO 2019-07-12 11:57:49,346 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 23111
INFO 2019-07-12 12:04:14,580 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=%27hdfc_bank%27 HTTP/1.1" 200 23141
ERROR 2019-07-12 12:05:01,410 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 68, in get_queryset
    % self.__class__.__name__
AssertionError: 'ShortBalanceSheetApi' should either include a `queryset` attribute, or override the `get_queryset()` method.
ERROR 2019-07-12 12:05:01,410 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 68, in get_queryset
    % self.__class__.__name__
AssertionError: 'ShortBalanceSheetApi' should either include a `queryset` attribute, or override the `get_queryset()` method.
ERROR 2019-07-12 12:05:01,411 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=%27hdfc_bank%27 HTTP/1.1" 500 105036
ERROR 2019-07-12 12:05:14,926 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 68, in get_queryset
    % self.__class__.__name__
AssertionError: 'ShortBalanceSheetApi' should either include a `queryset` attribute, or override the `get_queryset()` method.
ERROR 2019-07-12 12:05:14,926 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/generics.py", line 68, in get_queryset
    % self.__class__.__name__
AssertionError: 'ShortBalanceSheetApi' should either include a `queryset` attribute, or override the `get_queryset()` method.
ERROR 2019-07-12 12:05:14,926 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 500 104841
INFO 2019-07-12 12:05:33,052 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 23111
INFO 2019-07-12 12:06:22,727 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 22998
WARNING 2019-07-12 12:06:37,557 log --> Not Found: /api/v1/short-balance-sheet/hdfc_bank
WARNING 2019-07-12 12:06:37,557 log --> Not Found: /api/v1/short-balance-sheet/hdfc_bank
WARNING 2019-07-12 12:06:37,558 basehttp --> "GET /api/v1/short-balance-sheet/hdfc_bank HTTP/1.1" 404 2862
INFO 2019-07-12 12:06:55,361 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name HTTP/1.1" 200 23063
INFO 2019-07-12 12:07:03,906 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=yes_bank HTTP/1.1" 200 23106
ERROR 2019-07-12 12:11:47,499 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 154, in _get_response
    response = response.render()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/response.py", line 72, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 732, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 709, in get_context
    'filter_form': self.get_filter_form(data, view, request),
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 641, in get_filter_form
    html = backend().to_html(request, queryset, view)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django_filters/rest_framework/backends.py", line 103, in to_html
    template = loader.get_template(self.template)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: django_filters/rest_framework/form.html
ERROR 2019-07-12 12:11:47,499 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 154, in _get_response
    response = response.render()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/response.py", line 72, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 732, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 709, in get_context
    'filter_form': self.get_filter_form(data, view, request),
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 641, in get_filter_form
    html = backend().to_html(request, queryset, view)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django_filters/rest_framework/backends.py", line 103, in to_html
    template = loader.get_template(self.template)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: django_filters/rest_framework/form.html
ERROR 2019-07-12 12:11:47,500 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=yes_bank HTTP/1.1" 500 117837
ERROR 2019-07-12 12:11:51,075 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 154, in _get_response
    response = response.render()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/response.py", line 72, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 732, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 709, in get_context
    'filter_form': self.get_filter_form(data, view, request),
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 641, in get_filter_form
    html = backend().to_html(request, queryset, view)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django_filters/rest_framework/backends.py", line 103, in to_html
    template = loader.get_template(self.template)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: django_filters/rest_framework/form.html
ERROR 2019-07-12 12:11:51,075 log --> Internal Server Error: /api/v1/short-balance-sheet/
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 154, in _get_response
    response = response.render()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/response.py", line 72, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 732, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 709, in get_context
    'filter_form': self.get_filter_form(data, view, request),
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/rest_framework/renderers.py", line 641, in get_filter_form
    html = backend().to_html(request, queryset, view)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django_filters/rest_framework/backends.py", line 103, in to_html
    template = loader.get_template(self.template)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: django_filters/rest_framework/form.html
ERROR 2019-07-12 12:11:51,076 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=yes_bank HTTP/1.1" 500 117837
INFO 2019-07-12 12:15:16,799 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=yes_bank HTTP/1.1" 200 12319
INFO 2019-07-12 12:15:17,232 basehttp --> "GET /static/rest_framework/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 200 18028
INFO 2019-07-12 12:15:27,730 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=rbl_bank HTTP/1.1" 200 12311
INFO 2019-07-12 12:15:49,522 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=rbl_bank HTTP/1.1" 200 12311
INFO 2019-07-12 12:15:50,599 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=rbl_bank HTTP/1.1" 200 12311
INFO 2019-07-12 12:15:56,895 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=rbl_bank HTTP/1.1" 200 12311
INFO 2019-07-12 12:16:23,153 basehttp --> "GET /api/v1/short-balance-sheet/?year=2018-04-01 HTTP/1.1" 200 13585
INFO 2019-07-12 14:22:37,004 basehttp --> "GET /api/v1/short-balance-sheet/?year=2018-04-01 HTTP/1.1" 200 13585
WARNING 2019-07-12 14:23:04,484 log --> Bad Request: /api/v1/short-balance-sheet/
WARNING 2019-07-12 14:23:04,484 log --> Bad Request: /api/v1/short-balance-sheet/
WARNING 2019-07-12 14:23:04,484 basehttp --> "GET /api/v1/short-balance-sheet/?year=bob_bank HTTP/1.1" 400 9804
INFO 2019-07-12 14:23:18,331 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=bob_bank HTTP/1.1" 200 12411
WARNING 2019-07-13 15:18:08,839 log --> Not Found: /api/v1/balancesheet/
WARNING 2019-07-13 15:18:08,839 log --> Not Found: /api/v1/balancesheet/
WARNING 2019-07-13 15:18:08,840 basehttp --> "GET /api/v1/balancesheet/ HTTP/1.1" 404 2607
WARNING 2019-07-13 15:18:27,432 log --> Not Found: /api/v1/balancesheet/
WARNING 2019-07-13 15:18:27,432 log --> Not Found: /api/v1/balancesheet/
WARNING 2019-07-13 15:18:27,433 basehttp --> "GET /api/v1/balancesheet/?stock_name=hdfc_bank HTTP/1.1" 404 2628
WARNING 2019-07-13 15:18:49,600 log --> Not Found: /api/v1/short-balance-sheet//
WARNING 2019-07-13 15:18:49,600 log --> Not Found: /api/v1/short-balance-sheet//
WARNING 2019-07-13 15:18:49,600 basehttp --> "GET /api/v1/short-balance-sheet//?stock_name=hdfc_bank HTTP/1.1" 404 2652
INFO 2019-07-13 15:18:53,850 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 12420
INFO 2019-07-13 15:18:53,873 basehttp --> "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2019-07-13 15:18:53,874 basehttp --> "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2019-07-13 15:18:53,878 basehttp --> "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2019-07-13 15:18:53,879 basehttp --> "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2019-07-13 15:18:53,882 basehttp --> "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2019-07-13 15:18:53,884 basehttp --> "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2019-07-13 15:18:53,885 basehttp --> "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2019-07-13 15:18:53,938 basehttp --> "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2019-07-13 15:18:53,963 basehttp --> "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2019-07-13 15:18:54,006 basehttp --> "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2019-07-13 15:18:54,065 basehttp --> "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
INFO 2019-07-13 15:18:54,131 basehttp --> "GET /static/rest_framework/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 200 18028
INFO 2019-07-13 15:19:01,330 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 24138
INFO 2019-07-13 15:19:34,893 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 15:38:10,543 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 15:38:19,858 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 15:41:07,253 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:42:44,179 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:42:57,589 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:48:59,313 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:49:38,521 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:52:35,624 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:54:36,673 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:55:21,048 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:56:30,242 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:58:08,847 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:58:08,876 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=axis_bank HTTP/1.1" 200 2
INFO 2019-07-13 15:58:30,238 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:58:30,269 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=axis_bank HTTP/1.1" 200 2
INFO 2019-07-13 15:58:33,463 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:58:33,496 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=axis_bank HTTP/1.1" 200 2
INFO 2019-07-13 15:58:47,076 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:58:47,101 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=axis_bank HTTP/1.1" 200 2
INFO 2019-07-13 15:58:53,185 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:58:53,208 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=axis_bank HTTP/1.1" 200 2
INFO 2019-07-13 15:59:26,938 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:59:26,970 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=axis_bank HTTP/1.1" 200 2
INFO 2019-07-13 15:59:36,716 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 15:59:36,742 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=axis_bank HTTP/1.1" 200 2
INFO 2019-07-13 16:00:30,343 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 16:00:30,372 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=rbl_bank HTTP/1.1" 200 1018
INFO 2019-07-13 16:49:16,674 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 16:54:28,364 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 1034
INFO 2019-07-13 16:54:28,395 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=rbl_bank HTTP/1.1" 200 1018
INFO 2019-07-13 16:56:06,521 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 16:59:52,969 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 17:00:01,687 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 17:00:29,548 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 17:11:07,190 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 9283
INFO 2019-07-13 17:52:04,150 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:12:00,214 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:13:30,249 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:14:35,268 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:14:51,891 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:15:57,643 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:18:43,445 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:18:54,445 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-13 18:19:05,531 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 10083
INFO 2019-07-14 05:04:34,204 basehttp --> "GET /api/v1/short-balance-sheet/ HTTP/1.1" 200 12141
INFO 2019-07-14 05:28:01,203 basehttp --> "GET /api/v1/short-balance-sheet/?stock_name=hdfc_bank HTTP/1.1" 200 12428
INFO 2019-07-14 05:28:01,215 basehttp --> "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2019-07-14 05:28:01,219 basehttp --> "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
WARNING 2019-07-17 11:47:20,739 log --> Not Found: /api/v2/login/
WARNING 2019-07-17 11:47:20,739 log --> Not Found: /api/v2/login/
WARNING 2019-07-17 11:47:20,739 basehttp --> "GET /api/v2/login/ HTTP/1.1" 404 2653
ERROR 2019-07-17 11:47:22,689 log --> Internal Server Error: /
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 154, in _get_response
    response = response.render()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 81, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 63, in resolve_template
    return select_template(template, using=self.using)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/loader.py", line 47, in select_template
    raise TemplateDoesNotExist(', '.join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: home.html
ERROR 2019-07-17 11:47:22,689 log --> Internal Server Error: /
Traceback (most recent call last):
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/core/handlers/base.py", line 154, in _get_response
    response = response.render()
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 81, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/response.py", line 63, in resolve_template
    return select_template(template, using=self.using)
  File "/home/nandagopalparamasivam/PycharmProjects/Stock_Analysis/Django_Stock_Anlaysis/stock_env/lib/python3.6/site-packages/django/template/loader.py", line 47, in select_template
    raise TemplateDoesNotExist(', '.join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: home.html
ERROR 2019-07-17 11:47:22,690 basehttp --> "GET / HTTP/1.1" 500 90065
WARNING 2019-07-17 11:47:22,950 log --> Not Found: /favicon.ico
WARNING 2019-07-17 11:47:22,950 log --> Not Found: /favicon.ico
WARNING 2019-07-17 11:47:22,951 basehttp --> "GET /favicon.ico HTTP/1.1" 404 2647
INFO 2019-07-17 11:48:31,084 basehttp --> "GET / HTTP/1.1" 200 2
INFO 2019-07-17 11:52:17,861 basehttp --> "GET / HTTP/1.1" 200 10
INFO 2019-07-17 16:33:14,491 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:34:08,123 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:34:17,594 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:35:48,634 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:36:02,309 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:36:37,627 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:36:40,583 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:37:26,743 basehttp --> "GET / HTTP/1.1" 200 23
INFO 2019-07-17 16:39:49,659 basehttp --> "GET / HTTP/1.1" 200 10
WARNING 2019-07-18 05:35:10,385 log --> Not Found: /api/v2/login/
WARNING 2019-07-18 05:35:10,385 log --> Not Found: /api/v2/login/
WARNING 2019-07-18 05:35:10,385 basehttp --> "GET /api/v2/login/ HTTP/1.1" 404 2653
INFO 2019-07-18 05:35:12,986 basehttp --> "GET /swagger/ HTTP/1.1" 200 2230
INFO 2019-07-18 05:35:12,997 basehttp --> "GET /static/drf-yasg/style.css HTTP/1.1" 200 1047
INFO 2019-07-18 05:35:13,002 basehttp --> "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1" 200 154006
INFO 2019-07-18 05:35:13,003 basehttp --> "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1" 200 327334
INFO 2019-07-18 05:35:13,004 basehttp --> "GET /static/drf-yasg/insQ.min.js HTTP/1.1" 200 2093
INFO 2019-07-18 05:35:13,005 basehttp --> "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1" 200 944269
INFO 2019-07-18 05:35:13,007 basehttp --> "GET /static/drf-yasg/immutable.min.js HTTP/1.1" 200 56904
INFO 2019-07-18 05:35:13,015 basehttp --> "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1" 200 14715
INFO 2019-07-18 05:35:13,015 basehttp --> "GET /static/drf-yasg/url-polyfill.min.js HTTP/1.1" 200 6092
INFO 2019-07-18 05:35:14,940 basehttp --> "GET /swagger/?format=openapi HTTP/1.1" 200 1874
INFO 2019-07-18 05:35:14,942 basehttp --> "GET /static/drf-yasg/swagger-ui-dist/favicon-32x32.png HTTP/1.1" 200 628
